test/private-test.js

```diff
@@ -47,7 +47,10 @@ t.test('private functions', {autoend: true}, t => {
     const M3 = await clues(facts,'M3').catch(Object);
 
     t.same(MTOP,380,'is available indirectly');
-    t.same(M1.message,'M1 not defined','private fn not available directly');
+
+    // TODO: this isn't really possible... unless we always wrap in a promise?
```

Comment:
Should work if we wrap values from `$private` functions into promises?  Or we keep a hidden private registry on the parent logic.   Both ideas not ideal